app/pages/project/classify.cjsx

```diff
@@ -230,7 +230,7 @@ module.exports = React.createClass
 
   saveClassificationAndLoadAnotherSubject: ->
     @saveClassification()
-      .then @loadAnotherSubject()
+    @loadAnotherSubject()
 
```

Comment:
This is potentially dangerous, since loading a new subject destroys the current classification.

I think `saveClassification` creates a copy of the current classification, but I'm not sure.